docs/development.md

```diff
@@ -40,7 +41,20 @@ git clone git@github.com:ministryofjustice/laa-court-data-adaptor.git
 cd .../laa-court-data-adaptor
 ```
 
-- generate OAuth2 `client_credentials` - for the UI
+- configure local adaptor to use local mock common platform API, set database URL and, optionally, inline sidekiq jobs
+```
+# in .env.development.local
+COMMON_PLATFORM_URL=http://localhost:9293
+DATABASE_URL=postgres://localhost/laa_court_data_adaptor_development
+INLINE_SIDEKIQ=true
+```
+
+- setup CDA database
+```
+rails db:create db:migrate db:seed
+```
+
+- generate OAuth2 `client_credentials` - for the UI (see end of section for postgres error)
```

Comment:
@mabely not sure what is meant by see end of section for postgres error?

Reply:
@kmahern I've removed that, must have forgotten to remove by accident. I also added an alternative to setting up the db in the same commit so it aligns with CDA documentation, what I added didn't work for me but just so it doesn't diverge I included it - just FYI.